hpaction/tests/test_models.py

```diff
@@ -235,3 +237,22 @@ def test_clean_raises_when_case_type_is_not_chosen(self):
 
 def test_config_is_created_automatically(db):
     Config.objects.get()
+
+
+class TestBlankFields:
+    def test_harassment_details(self, db):
+        user = UserFactory()
+        hd = HarassmentDetails(user=user)
+        hd.full_clean()
+
+    def test_fee_waiver_details(self, db):
+        user = UserFactory()
+        fw = FeeWaiverDetails(
+            user=user, income_amount_monthly=Decimal("3.15"), rent_amount=Decimal("2.15")
+        )
+        fw.full_clean()
+
+    def test_hpaction_details(self, db):
+        user = UserFactory()
+        hpad = HPActionDetails(user=user)
+        hpad.full_clean()
```

Comment:
I added these tests to make sure that all the `NullBooleanField`s in the original model still defaulted to `null`, rather than raising validation errors about not being filled out.